src/Microsoft.Extensions.Testing.Abstractions/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using Microsoft.DotNet.ProjectModel;
+
+namespace Microsoft.Extensions.Testing.Abstractions
+{
+    public static class ProjectContextExtensions
```

Comment:
This can totally live in `Microsoft.DotNet.ProjectModel`, it's general enough IMO. Also, we can have `Get` prefix on these methods since they are methods, not properties (*sigh* extension properties :)).